Jd pull requests - Added remaining sections of the page

JuanDiegoGuerra opened the pull request:
Updated index.html / mobile_first.css
Completed the sections that were missing from the main template. Fixing the styles for those sections and added one last img for the last section background.

Please, let me know if I missed to add something or if I did great. It's been plenty of hours of coding these days and I learned a lot but didn't sleep enough jeje 😅.

I'll patiently wait for a review.

JuanDiegoGuerra committed:
Update index.html

Updated index.html adding the "my-recent-work" section to implement the projects that I want to show in the portfolio.

JuanDiegoGuerra committed:
Update mobile_first.css

Updated mobile_first.css to implement the design of the "my-recent-work" section. Also added 2 imgs for it and 1 img for "about-me" section.

JuanDiegoGuerra committed:
Update index.html

Added about-me section with its subsections (languages-frameworks-skills)

JuanDiegoGuerra committed:
Update mobile_first.css

Added the styles needed for the new section's design.
Also added some images for it and deleted one that wasn't neccessary anymore.

JuanDiegoGuerra committed:
Update index.html

Added last section of the page (footer contact-me)

JuanDiegoGuerra committed:
Update mobile_first.css

Added all the styles needed to give style to the last section (footer contact-me) and the rest of the project.
Added 1 img for last section bg.

JuanDiegoGuerra committed:
Update mobile_first.css

Fixed some hierarchy errors.

Shedrack-Sunday commented on index.html:
```diff
@@ -41,11 +41,144 @@ <h2 class="headline-sub-title">I'm a Micronout</h2>
                </footer>
             </section>
     </div>
+    <div id="my-recent-works">
```
- [ ] For this section, kindly make the container of the section a `section` tag. This is in line of having a semantically well coded project and helps keep your project professional

Shedrack-Sunday commented on css/mobile_first.css:
```diff
@@ -116,22 +135,392 @@ body {
+  margin-top: -28px;
+  text-align: center;
+  align-content: center;
+}
```
- [ ] The main container of the projects should have a `display:grid` property. This was stated in the project requirements, and made so to help you understand how to use the `grid` property to display and structure content 👍 

## Instruction:
![image](https://user-images.githubusercontent.com/50990925/227669751-9d9cf5e6-ed9a-413b-95c8-3c02815624fc.png)

Shedrack-Sunday commented on index.html:
```diff
@@ -41,11 +41,144 @@ <h2 class="headline-sub-title">I'm a Micronout</h2>
             </section>
     </div>
+    <div id="my-recent-works">
+        <h1 class="recent-works-tittle">My Recent <br> Works</h1>
```
- [  ] Only one `h1` tag should be used per project. This is in line with using the right semantics, which directly affects how the web crawls your website for example. So kindly use a `hx` element like `h2` `h3` for this section 👍

Shedrack-Sunday commented on css/mobile_first.css:
```diff
@@ -116,22 +135,392 @@ body {
+  border-radius: 8px;
+  border: 0.5px solid #d0d9d4;
+  text-align: center;
+}
```
- [ ] Kindly make sure the projects listed are aligned with the flexbox. The use of the flex display was introduced to help you understand how to effectively use such which is an industry best practice

Shedrack-Sunday commented on index.html:
```diff
@@ -41,11 +41,144 @@ <h2 class="headline-sub-title">I'm a Micronout</h2>
+                Hello-Microverse-Project <br>
+                First Week
+              </h3>
+                <div class="languages-grid">                    
+                        <div class="lang-list l1"><strong>Html5</strong></div>
+                        <div class="lang-list l2"><strong>Css3</strong></div>
+                        <div class="lang-list l3"><strong>Git</strong></div>
+                        <div class="lang-list l4"><strong>Github</strong></div>
+                </div>
```
- [ ] For this section, the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") is a list `ul` and each of them a `li`

## Instructions:
![image](https://user-images.githubusercontent.com/50990925/227673145-0c623600-cc8f-41b7-801f-ba8443f29116.png)

Shedrack-Sunday commented on index.html:
```diff
@@ -41,11 +41,144 @@ <h2 class="headline-sub-title">I'm a Micronout</h2>
+                Setup-Portfolio-Mobile-1st <br>
+                First Week
+              </h3>
+                <div class="languages-grid">                    
+                        <div class="lang-list l1"><strong>Html5</strong></div>
+                        <div class="lang-list l2"><strong>Css3</strong></div>
+                        <div class="lang-list l3"><strong>Git</strong></div>
+                        <div class="lang-list l4"><strong>Github</strong></div>
```
- [ ] Please refer to line `63` on the use of proper semantics for this section 👍

Shedrack-Sunday commented on index.html:
```diff
@@ -41,11 +41,144 @@ <h2 class="headline-sub-title">I'm a Micronout</h2>
+              <h3 id="h3-lang">
+                Languages
+              </h3>
+                <div class="am-languages-grid">                    
+                        <div class="am-lang-list am1"><strong>Html5</strong></div>
+                        <div class="am-lang-list am2"><strong>Css3</strong></div>
+                        <div class="am-lang-list am3"><strong>Git</strong></div>
+                        <div class="am-lang-list am4"><strong>Github</strong></div>
```
- [ ] Please refer to line `63` on the use of proper semantics for this section 👍

Shedrack-Sunday commented on index.html:
```diff
@@ -41,11 +41,144 @@ <h2 class="headline-sub-title">I'm a Micronout</h2>
+                  <h3 id="h3-lang">
+                    Frameworks
+                  </h3>
+                    <div class="am-frameworks-grid">                    
+                            <div class="am-frame-list f1"><strong>Angular</strong></div>
+                            <div class="am-frame-list f2"><strong>React</strong></div>
+                            <div class="am-frame-list f3"><strong>Spring</strong></div>
+                            <div class="am-frame-list f4"><strong>ASP.NET</strong></div>
+                            <div class="am-frame-list f5"><strong>NodeJS</strong></div>
```
- [ ] Please refer to line `63` on the use of proper semantics for this section 👍

Shedrack-Sunday commented on index.html:
```diff
@@ -41,11 +41,144 @@ <h2 class="headline-sub-title">I'm a Micronout</h2>
+                  <h3 id="h3-lang">
+                    Skills
+                  </h3>
+                    <div class="am-skills-grid">
+                            <div class="am-frame-list sk1"><strong>Database Management</strong></div>                    
+                            <div class="am-frame-list sk2"><strong>Version Control</strong></div>
+                            <div class="am-frame-list sk3"><strong>Web Development</strong></div>
+                            <div class="am-frame-list sk4"><strong>UX/UI</strong></div>
+                            <div class="am-frame-list sk5"><strong>SQL</strong></div>
```
- [ ] Please refer to line `63` on the use of proper semantics for this section 👍

Shedrack-Sunday commented on index.html:
```diff
@@ -41,11 +41,144 @@ <h2 class="headline-sub-title">I'm a Micronout</h2>
+                </div>
+            </div>
+        </div>       
+    </div>
+    <div id="about-me">
+        <div class="about-me-bg">
```
- [ ] For this section, The container of the section should be a  `section` tag. This is semantically correct as a best practice, and helps the crawling of your webpage easier 👍

Shedrack-Sunday commented on index.html:
```diff
@@ -41,11 +41,144 @@ <h2 class="headline-sub-title">I'm a Micronout</h2>
+    </div>
+    <div id="about-me">
+        <div class="about-me-bg">
+        <h1 class="about-me-tittle">About me</h1>
```
- [ ] Using only one `H1` creates a hierarchy and gives your page a more logical structure. It makes it easy for readers (and those using screen readers) to skim and understand what your page is about. Kindly use a `hx` instead 👍

Shedrack-Sunday commented on css/mobile_first.css:
```diff
@@ -116,22 +135,392 @@ body {
   margin-top: 5%;
 }
 
+.icon-2 {
+  list-style-type: none;
+  margin-right: 10%;
+}
+
 h3 {
   margin-top: 12px;
   margin-bottom: 0;
   font-weight: bold;
-  font-size: 32px;
+  font-size: 18px;
+  color: var(--gris);
+  line-height: 25px;
+}
+
+#h3-lang {
+  margin-top: 0;
+  margin-bottom: 0;
+  font-weight: bold;
+  font-size: 18px;
   color: var(--gris);
-  line-height: 44px;
+  line-height: 25px;
 }
 
-footer h3 {
-  font-style: normal;
+#my-recent-works {
```
- [ ] Kindly adjust the margin at the top of the works section to look visually appealing and give a better user experience. This is also important in sticking to requirements 

## Your design:
![image](https://user-images.githubusercontent.com/50990925/227675397-42f714f2-beae-4e87-a047-fd9b31858ba2.png)

## Figma:
![image](https://user-images.githubusercontent.com/50990925/227675422-96a6da0e-a729-4c56-baaa-f4bc659cc1d8.png)

Shedrack-Sunday reviewed (changes requested):
Hi @JuanDiegoGuerra      👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 
![](https://user-images.githubusercontent.com/80612925/157397398-00904e26-60ec-46cd-9510-240661c250ef.gif)
  
# Required Changes ♻️

- [x] I must admit you have really put in the effort so far in this project, however, the changes required below need to be implemented which majorly has to do with semantics and a bit of styling 


_check the comment section for a detailed review_ 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

JuanDiegoGuerra committed:
Update index.html / mobile_first.css

Fixed what was requested in my last review.
-Added "section" tag where needed.
-Fixed sections that had to be displayed as grid and then use flexbox
-Deleted repetead "h1" tag elements, edited to be h2-h5
-Fixed css with the section that were mention before and now are functioning with flexbox.
-Fixed index.html in where some sections I used "div" tags instead of "ul" + "li" tags.

LionRouge1 reviewed (approved):
Hi @JuanDiegoGuerra, 

# STATUS: APPROVED! 🥇 💯 
Great work on making the changes 👏 
You've done well implementing the requested changes
![done](https://media.giphy.com/media/l1uga8ObPBnIBoXUQ/giphy.gif)

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

[OPTIONAL]
- [ ] Please You must stick to the design as much as possible (e.g., font, colors, images, text, margins). Your website background color and font are different from the design color.

### Your website
![Screenshot from 2023-03-25 08-14-28](https://user-images.githubusercontent.com/91142432/227706167-1e65f954-907b-4437-a19c-fca3190df429.png)

### How it suppose to look like
![227675422-96a6da0e-a729-4c56-baaa-f4bc659cc1d8](https://user-images.githubusercontent.com/91142432/227706268-24415dbf-f061-4926-9cc1-cb1e8bede391.png)

This is a requirement of the project but the first reviewer didn't notice it. so I requested this as an optional but I highly recommend you to make the change.

****You can now get some sleep, because you deserve it**  :sunglasses:**   🎉 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

JuanDiegoGuerra commented:
@LionRouge1 Hello! Thank you for your feedback, I just woke up and I'm happy to read that I achieved the goal. I will also try to improve what you suggested me. I tried to make it as similar as possible and maybe in my perspective (screen sizes tested) looked good and I didn't notice the differences but I know I can do it better next time!

![thank_you_kr](https://user-images.githubusercontent.com/45726631/227732106-be1d222f-e00a-44eb-ae82-892988b75719.gif)
